pkg/operator/ceph/object/spec.go

```diff
@@ -310,6 +311,20 @@ func (c *clusterConfig) makeDaemonContainer(rgwConfig *rgwConfig) v1.Container {
 			)
 		}
 	}
+	if c.store.Spec.Security != nil && c.store.Spec.Security.OpenPolicyAgentService != nil {
+		container.Args = append(container.Args,
+			cephconfig.NewFlag("rgw use opa authz", "true"),
+			cephconfig.NewFlag("rgw opa url",
+				c.store.Spec.Security.OpenPolicyAgentService.URL),
+			cephconfig.NewFlag("rgw opa token",
+				c.store.Spec.Security.OpenPolicyAgentService.Token),
+		)
+		if c.store.Spec.Security.OpenPolicyAgentService.VerifySSL != nil {
+			container.Args = append(container.Args,
+				cephconfig.NewFlag("rgw opa verify ssl",
+					strconv.FormatBool(*c.store.Spec.Security.OpenPolicyAgentService.VerifySSL)))
```

Comment:
Is the bool pointer really necessary? If it's not required why not just let the default. `false`?